asr/logging.py

```diff
@@ -0,0 +1,190 @@
+import json
+from datetime import datetime
+from pathlib import Path
+
+
+import numpy as np
+
+
+def read_log(log_fp):
+    """Read log file.
+
+    Arguments
+    ---------
+    log_fp: str, pathlib.Path
+        File path of the log file.
+
+    Returns
+    -------
+    Logger:
+        A Logger object with logs.
+    """
+    log_fp_path = Path(log_fp)
+
+    if log_fp_path.is_file() and log_fp_path.suffix == ".json":
```

Comment:
I don't think checking for suffix should be necessary? Just try/except when loading the file?

Reply:
Great! Will change this one.